File: SumOfPairs/SumOfPairs.py
def SumOfPairs(arr , sum):
    count = 0
    for i in range(len(arr)-1):
        for j in range(i+1,len(arr), 1):
           if sum  == arr[i] + arr[j]:
               count +=1
    print(count)  

SumOfPairs([1, 5, 7, 1], 6)

File: SumOfPairs/SumOfPairs.js
function SumOfPairs(arr , sum){
    let count = 0;
    for(i= 0 ; i < arr.length -1 ; i++){
        for(j = i+1 ; j  < arr.length; j++){
            if(arr[i]+ arr[j] == sum){
                count++
            }
        }
    }
    console.log(count)
}

SumOfPairs([1, 5, 7, 1], 6)